### PR TITLE
Zer 139 update mob UI calculator

### PR DESCRIPTION
**Pull request by Beerka1**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @opazyniuk 

### Second Level Review

- [ ] @PavloMS 

## Summary of issue

1) In the mobile version: the logo should be aligned with the blocks on the left edge, the "Support" buttons and the language selection button should be placed one above the other and aligned with the blocks on the right edge.

2) In the mobile version: the text of the data entry block must be located on the same line as the data entry lines (on the left edge).

3) In the mobile version: the data input block and the result display block must be of the same width.

## Summary of change

Before

Width: 767
image

Width: 575
image

After

Width: 767
image

Width: 575
image

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by malk1ss — APPROVED** [review]

**Review by anastasiiahryniv — APPROVED** [review]

File: Gemfile.lock [hunk 1]
```diff
@@ -96,7 +96,7 @@ GEM
     childprocess (4.1.0)
     cocoon (1.2.15)
     coderay (1.1.3)
-    concurrent-ruby (1.1.9)
+    concurrent-ruby (1.1.10)
```
Comment (opazyniuk) [inline Gemfile.lock:99]: remove changes form gemfile

**Review by opazyniuk — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "@rails/actioncable": "^6.0.0",
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
-    "@rails/webpacker": "5.4.3",
+    "@rails/webpacker": "^5.4.3",
```
Comment (opazyniuk) [inline package.json:13]: why do you add this changes?

File: yarn.lock [hunk 3]
```diff
@@ -1582,9 +1582,9 @@
   resolved "https://registry.npmjs.org/@rails/ujs/-/ujs-6.1.4.tgz"
   integrity sha512-O3lEzL5DYbxppMdsFSw36e4BHIlfz/xusynwXGv3l2lhSlvah41qviRpsoAlKXxl37nZAqK+UUF5cnGGK45Mfw==
 
-"@rails/webpacker@5.4.3":
+"@rails/webpacker@^5.4.3":
```
Comment (opazyniuk) [inline yarn.lock:1585]: why do you add this changes?

**Review by opazyniuk — CHANGES_REQUESTED** [review]

**Review by opazyniuk — CHANGES_REQUESTED** [review]
please add screenshots with before/after your changes results

File: package.json (hunk 2 above)
Comment (Beerka1) [reply package.json]: Not needed, reverted it.

File: yarn.lock (hunk 3 above)
Comment (Beerka1) [reply yarn.lock]: Not needed, reverted it.

**Review by opazyniuk — APPROVED** [review]

**Pull request merged**